tests/commands/export.py

```diff
@@ -31,9 +32,52 @@ def test_export_project(capfd, afrikaans_tutorial, french_tutorial):
 
 @pytest.mark.cmd
 @pytest.mark.django_db
+def test_export_project_and_lang(capfd, afrikaans_tutorial, french_tutorial):
```

Comment:
@phlax can you give me a quick lesson then, I'm not sure I understand the new fixtures